lua/entities/acf_glatgm/init.lua

```diff
@@ -197,7 +197,7 @@ function ENT:Think()
 	local Position  = self.Position
 
 	if not IsDelayed and IsValid(Computer) then
-		local StartPos = Computer:LocalToWorld(Computer.Offset)
+		local StartPos = Computer:LocalToWorld(Computer.Offset or Vector(6, -1, 0))
```

Comment:
Weird, maybe the computers aren't storing this information when they're created/updated. I'll have to remember to dig into this later, but for now it's an acceptable patch